aziotctl/aziotctl-common/src/restart.rs

```diff
@@ -0,0 +1,59 @@
+use std::io::{self, Write};
+use std::process::Command;
+
+use crate::ServiceDefinition;
+
+pub fn restart(services: &[&ServiceDefinition]) {
+    // stop all services
+    for ServiceDefinition {
+        service,
+        sockets: _,
+    } in services
+    {
+        print!("Stopping {}...", service);
+        let result = Command::new("systemctl")
+            .args(&["stop", service])
+            .output()
+            .unwrap();
+
+        if result.status.success() {
+            println!("Stopped!");
+        } else {
+            println!("\nError stopping {}", service);
```

Comment:
suggestions: switch this to `eprintln!`, and write out the `result.stdout` and `result.stderr` to `stdout`. You might also want to add a `eprintln!` between lines 23 and 24 that breaks-up the process's stdout and stderr.